server.js

```diff
@@ -20,13 +21,106 @@ mongoose.Promise = Promise;
 const port = process.env.PORT || 8080;
 const app = express();
 
+//Mongoose model// 
+const Book = mongoose.model('Book', {
+  bookID: Number,
+  title: String,
+  authors: String,
+  average_rating: Number,
+  isbn: Number,
+  isbn13: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+})
+
+//Loop through data set. Async in order to avoid duplication.
+if (process.env.RESET_DB) {
+  const seedDatabase = async () => {
+    await Book.deleteMany() 
+    booksData.forEach(singleBook => {
+      const newBook = new Book(singleBook)
+      newBook.save()
+    })
+  }
+  seedDatabase()
+}
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
 // Start defining your routes here
-app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+    app.get('/', (req, res) => {
+      res.send(
+        {"Welcome":"This is the ultimate API source for book reviews!",
```

Comment:
Good job with the welcome page indicating the endpoints used and everything works perfectly. :)